src/mudlet-lua/tests/GUIUtils_spec.lua

```diff
@@ -266,3 +266,8 @@ describe("Tests the GUI utilities as far as possible without mudlet", function()
     end)
   end)
 end)
+
+--[[
+  TODO:
+    replaceLine and variants
```

Comment:
Lazy 😛

Reply:
Little bit. I'm struggling with a way to stub the test out that doesn't render it essentially useless/cause it to test the implementation details rather than the functionality.  Really need to get the c/d/hreplace functions tested, as the *Line variants just selectCurrentLine first.